### DISCUSSION TITLE
Notify-Container and UnifiedPush Provider-App: Interferance?

### DISCUSSION THREAD

**Discussion by maste9** [nextcloud/all-in-one#3978] [Questions] [answered]
Solved. tl;dr [Migration is completely fine but one has to remove the registered device after migration and reregister it.](https://codeberg.org/NextPush/nextpush-android/issues/4#issuecomment-1411868)

> I migrated yesterday (successfully - at last :sweat_smile: ) from a not supported docker-compose install to AIO.
> 
> I used the __UnifiedPush Provider__-App to connect Android-Devices with the NextPush-App with my old setup. Which worked fine.
> 
> Since the migration yesterday we receive "5 disconnects"-warnings from the Android-App, ~which I might have solved by setting the reverse proxy to 10m timeout limits as https://github.com/UP-NextPush/server-app recommends. :crossed_fingers:~
> No, the disconnects come because the App doesn't reach a working API. Even localhost:11000 can't reach https://codeberg.org/NextPush/uppush/raw/branch/main/script/tests.sh So that's obviously not a problem of my npm reverse proxy as I thought before.
> 
> But while fixing that I wondered if the NotifyPush-Container and the NextPush-App do actually interfere in any way.
> 
> And if they worked independently of each other - do I need both?  Or is it a redundancy?

**Comment by szaimen** [comment, marked as answer]
I think this is solved in the meantime
